builtin/mainmenu/serverlistmgr.lua

```diff
@@ -153,6 +153,14 @@ local function fetch_geoip()
 end
 
 function serverlistmgr.sync()
+	if minetest.settings:get("serverlist_lan") then
+		if core.ask_lan_servers then --This checks if the function exists before running it.
+			core.ask_lan_servers();
+		else
+			print("core.ask_lan_servers isn't defined.")
+		end
+	end
+
```

Comment:
why wouldn't it be defined?

Reply:
As far as I know, it would be defined either way, returning null if the setting isn't enabled and returning whatever it finds if it is enabled, this is probably still here from when I was having trouble registering the function properly, this shouldn't be necessary in the final version.